pkg/kudoctl/kudoinit/crd/crds.go

```diff
@@ -36,6 +36,10 @@ func NewInitializer() Initializer {
 	}
 }
 
+func (c Initializer) Description() string {
```

Comment:
totally agree this should be Stringer interface... 

I'm not exactly sure how we got to the "Initializer` struct in the first place.    The `InitStep` also feels completely off.   An interface would be something you would use more than once and would be reduced in size.   If this was a `Step` I might be able to see it... but an `InitStep`.  How many and how many kinds of InitSteps are expected?